DSharpPlus/Entities/Interaction/DiscordInteraction.cs

```diff
@@ -90,6 +90,37 @@ public DiscordChannel Channel
     /// </summary>
     [JsonProperty("guild_locale")]
     public string? GuildLocale { get; internal set; }
+    
+    /// <summary>
+    /// The permissions allowed to the application for the given context.
+    /// </summary>
+    /// <remarks>
+    /// For guilds, this will be the bot's permissions. For group DMs, this is `ATTACH_FILES`, `EMBED_LINKS`, and `MENTION_EVERYONE`.
+    /// In the context of the bot's DM, it also includes `USE_EXTERNAL_EMOJI`.  
+    /// </remarks>
+    [JsonProperty("app_permissions", NullValueHandling = NullValueHandling.Ignore)]
+    public Permissions AppPermissions { get; internal set; }
+    
+    /// <summary>
+    /// Gets the interactions that authorized the interaction.
+    ///     <para>
+    ///         This dictionary contains the following:
+    ///             - If the interaction is installed to a user, a key of <see cref="ApplicationIntegrationType.User"/> and a value of the user's ID.
+    ///             - If the interaction is installed to a guild, a key of <see cref="ApplicationIntegrationType.Guild"/> and a value of the guild's ID.
+    ///                 - IF the interaction was sent from a guild context, the above holds true, otherwise the ID is 0. 
+    ///     </para>
+    /// </summary>
```

Comment:
this xmldoc does not work, you should instruct it specifically with `<br/>` or list tags to become a list